src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -676,4 +681,16 @@ private void UpdateShownPatchDetails()
         // Enable move to patch button if this is a valid move
         moveToPatchButton.Disabled = !editor.IsPatchMoveValid(patch);
     }
+
+    private void OnSpeciesNameTextChanged(string newText)
```

Comment:
This should probably be called from where the current species name is set, as I read in the godot documentation that directly setting the text doesn't cause the text changed signal?